client/my-sites/email/form/mailboxes/components/utilities/mailbox-operations.ts

```diff
@@ -22,7 +22,7 @@ export class MailboxOperations {
 				.map( ( mailbox ) => mailbox.getFieldValue< string >( FIELD_MAILBOX ) ?? '' );
 
 			mailbox.validate(
-				true,
+				false,
```

Comment:
Let's revert this change since the `hiddenFieldNames` prop controls which fields are hidden and the `usePasswordResetEmailField` hook unhides the password reset email field if the default value is invalid.